Update vb6-support.md

danru-ms opened the pull request:
# Title
Update vb6-support.md to remove text saying "Custom Support Agreements may be available".

## Summary
Update vb6-support.md to remove text saying "Custom Support Agreements may be available".

## Details
Custom Support Agreements are not available for Visual Basic 6 IDE.

## Suggested Reviewers
paulyuk@microsoft.com

Commit:
Update vb6-support.md

Remove text saying Custom Support Agreements may be available.

dnfclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

mairaw commented:
@paulyuk you were requested as a reviewer here. Can you take a look? Thx!

mairaw reviewed (commented):
LGTM. It would be good to change the date on line 3 to reflect this topic has been updated recently.

Commit:
Update vb6-support.md

Update text to remove Windows versions that are out of mainstream and extended support (Windows Vista, 8, Server 2003).  Add Windows Server 2016 to table per reader feedback.  Simplify title and section titles to improve article readability.

Commit:
Update vb6-support.md

Update article revision date, add link to Windows lifecycle fact sheet.

svick commented on docs/visual-basic/reference/vb6-support.md:
```diff
@@ -14,21 +14,21 @@ ms.assetid: ffc5ba4d-44d7-4ef7-a3f6-38a8738bf127
-# Support Statement for Visual Basic 6.0 on Windows Vista, Windows Server 2008, Windows 7, Windows 8 and Windows 8.1, Windows Server 2012, Windows 10, and Windows Server 2016
+# Support Statement for Visual Basic 6.0 on Windows
 
-Update: March 2016
```
Is it necessary to specify the update date twice? Currently, the start of the article looks like this:

![](https://user-images.githubusercontent.com/287848/29810375-d749fe20-8c9f-11e7-8c8f-ac67f8c0bc56.png)

mairaw replied:
No, we should remove this line

danru-ms replied:
Thanks @svick @mairaw!  Removed in latest commit to the PR.

svick commented:
This article doesn't seem to be correctly placed in the TOC. Is that intentional?

mairaw commented:
@svick I've noticed that too. Including @rpetrusha here since he brought the topic over and maybe knows whether it is by design or not.

rpetrusha commented:
@svick @mairaw Yes, it's by design. Since it's a legacy topic, it's available only from links on support topics.

Commit:
Update vb6-support.md

Removed "Update: August 2017" per feedback since we have an edited date.

danru-ms commented:
@mairaw I believe this is ready to pull.  LMK if you have any other feedback.

mairaw reviewed (approved):
LGTM 👍
